languagetool-commandline/src/main/java/org/languagetool/commandline/CommandLineParser.java

```diff
@@ -222,6 +225,7 @@ public void printUsage(PrintStream stream) {
             + "                           ngram occurrence counts; activates the confusion rule if supported\n"
             + "  --word2vecmodel DIR      a directory with e.g. 'en' sub directory (i.e. a language code) that contains\n"
             + "                           final_embeddings.txt and dictionary.txt; activates neural network based rules\n"
+            + "  --neuralnetworkmodel DIR a base directory for various saved neural network models\n"
```

Comment:
Thanks for the work!  I could find an introduction for [--word2vecmodel](https://github.com/gulp21/languagetool/tree/neuralnetworkrule), but how to use `--neuralnetworkmodel`? What kind of file type it is expected?